bins/packages/hd-idle/hd-idle.sh

```diff
@@ -0,0 +1,35 @@
+HDIDLE_VERSION="master"
+HDIDLE_REPOSITORY="https://github.com/adelolmo/hd-idle.git"
+
+dependencies_hd-idle() {
+    . ${PKGDIR}/../golang/golang.sh
+    build_golang
+    HD_IDLE_HOME="${GOPATH}/src/github.com/adelolmo"
+}
+
+download_hd-idle() {
+    echo "[+] downloading and installing hd-idle"
+    go get github.com/adelolmo/hd-idle
```

Comment:
We are supposed to ship specific version and not latest version automatically, right @zaibon ?
You should download a specific version from github, not using `go get` like this.

Reply:
right specific version are better so we control exactly what run. Otherwise we can have different version running on different node and this makes a mess. 
But with go mod, now you can control the exact version of what you build, so something like `go get github.com/adelolmo/hd-idle@v1.2.3`

Reply:
@zaibon looks like that is not possible =/

```
➜  test go get github.com/adelolmo/hd-idle@v1.9     
go get github.com/adelolmo/hd-idle@v1.9: no matching versions for query "v1.9"
```